### PR TITLE
docs : Added setup tutorial link

### LINKED ISSUE #277: Docs :  Add setup tutorial to `.github/ENV_SETUP_INSTRUCTION.md`
### Description

- I want to add an environment setup video link to `ENV_SETUP_INSTRUCTION.md`
- This video tutorial helps new contributors if they got stuck while setting up a dev environment.

### PR DESCRIPTION
**Pull request by Aryamanz29**
### Description
- Added setup tutorial link to `.github/ENV_SETUP_INSTRUCTION.md`

Fixes #277

### Type of Change:

- Documentation

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by Aryamanz29** [comment]
@mtreacy002  Please review this PR

File: .github/ENV_SETUP_INSTRUCTION.md
Comment (vj-codes) [inline .github/ENV_SETUP_INSTRUCTION.md:172]: keep this and remove hr tag as it was before

File: .github/ENV_SETUP_INSTRUCTION.md
Comment (vj-codes) [inline .github/ENV_SETUP_INSTRUCTION.md]: don't use strong tag, use `**text here**` instead

**Review by vj-codes — CHANGES_REQUESTED** [review]
Thank you for the video, please look at the comments @Aryamanz29

**Comment by Aryamanz29** [comment]
@vj-codes  Please review it again 😅

**Review by vj-codes — DISMISSED** [review]
@Aryamanz29 thank you for your contribution , LGTM 🎉 
However, do wait for Maya's final review :)

File: .github/ENV_SETUP_INSTRUCTION.md
Comment (mtreacy002) [inline .github/ENV_SETUP_INSTRUCTION.md]: @Aryamanz29 . This looks good to me already. What do you think if we take it a little step further by embedding the youtube image in here instead of just url link? [Here](https://www.nathan-lam.com/blog/embed-video-in-github-readme.html)'s an article that gives various options on how to do this. I'm happy to whichever one you choose from the options 😉 .

**Review by mtreacy002 — COMMENTED** [review]
Hi @Aryamanz29 , thank you so much for adding your tutorial video here. I'm happy to approve it as it is, but want to check if you're happy to take it a bit further as per my feedback below. Let me know what you think 😉

File: .github/ENV_SETUP_INSTRUCTION.md
Comment (Aryamanz29) [reply .github/ENV_SETUP_INSTRUCTION.md]: @mtreacy002  Done 👍 , I've used a youtube thumbnail image.

**Review by mtreacy002 — APPROVED** [review]
LGTM, @Aryamanz29 🎉 🎉 🎉 . Thank you so much for accommodating for my suggestion 😉 . I'm approving this now.

**Pull request merged**
